src/calculateRentalCost.js

```diff
@@ -5,6 +5,28 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const daylyRent = 40;
+  let amount = days * daylyRent;
+
+  if (days <= 3) {
+    amount = days * daylyRent;
+
+    // return amount;
+  }
+
+  if (days >= 3 && days < 7) {
+    amount = days * daylyRent - 20;
+
+    return amount;
+  }
+
+  if (days >= 7) {
+    amount = days * daylyRent - 50;
+
+    // return amount;
+  }
```

Comment:
Instead of using `if (days >= 7)`, you can simplify the condition by using `else if (days >= 7)` to make it clear that this block is mutually exclusive with the previous ones.